src/table.js

```diff
@@ -666,3 +762,118 @@ export function __table(source, operations) {
   }
   return source;
 }
+
+function coerceRow(object, types) {
+  let coerced = {};
+  for (var key in object) {
```

Comment:
We shouldn’t use a for-in here; we should be using the schema to know which columns are defined (and in which order).